testcases/open_posix_testsuite/conformance/interfaces/pthread_create/14-1.c

```diff
@@ -76,8 +79,10 @@ static void *sendsig(void *arg)
 		 * can progress.
 		 */
 		sleep_time++;
-		if (sleep_time / SIGNALS_WITHOUT_DELAY > 0)
-			usleep(sleep_time / SIGNALS_WITHOUT_DELAY);
+		if (sleep_time / SIGNALS_WITHOUT_DELAY > 0) {
+			time_between_signals_ts.tv_nsec = sleep_time / SIGNALS_WITHOUT_DELAY;
```

Comment:
Should this be '* 1000' ? Original sleep_time is in microseconds.

Reply:
Yes! Good catch :).